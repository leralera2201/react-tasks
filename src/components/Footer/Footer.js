import React from 'react';
import { Navbar } from 'reactstrap';
import { Link } from 'react-router-dom';

const Footer = () => (
  <Navbar color="dark" dark expand="md" className="flex-center h50">
    <Link to="/" className="fz30 color-f">
      Blog
    </Link>
  </Navbar>
);

export default Footer;
