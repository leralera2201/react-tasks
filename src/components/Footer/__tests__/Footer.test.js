import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Footer from "../Footer";

describe("Testing Footer component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
