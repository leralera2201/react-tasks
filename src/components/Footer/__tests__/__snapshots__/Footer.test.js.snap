// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer component renders correctly 1`] = `
<Navbar
  className="flex-center h50"
  color="dark"
  dark={true}
  expand="md"
  tag="nav"
>
  <Link
    className="fz30 color-f"
    to="/"
  >
    Blog
  </Link>
</Navbar>
`;
